internal/pkg/generate/generate_scripts.go

```diff
@@ -325,6 +360,13 @@ func ComposeFile(gd *GenData, at io.Writer) error {
 	return nil
 }
 
+func (d DockerComposeData) DistributedValidatorEndpoint() string {
+	if d.Distributed {
+		return configs.OnPremiseDistributedValidatorURL
+	}
+	return ""
+}
+
```

Comment:
This is not being used. If you are not planning on using it, please remove it.

Reply:
Removed, thanks!